### PR TITLE
Support org level rbac

### PR DESCRIPTION
**Pull request by motatoes**

### CODE REVIEW

File: pkg/policy/policy.go [hunk 1]
```diff
@@ -22,43 +23,91 @@ type DiggerHttpPolicyProvider struct {
...
 }
 
-func (p NoOpPolicyChecker) Check(_ string, _ string, _ interface{}) (bool, error) {
+func (p NoOpPolicyChecker) Check(_ string, _ string, _ string, _ interface{}) (bool, error) {
 	return true, nil
 }
 
-func (p *DiggerHttpPolicyProvider) GetPolicy(namespace string, projectName string) (string, error) {
-	// fetch RBAC policies from Digger API
+func (p *DiggerHttpPolicyProvider) getPolicyForOrganisation(organisation string, projectName string) (string, *http.Response, error) {
```
Comment (Spartakovic) [inline pkg/policy/policy.go]: don't need a projectname probably

Comment (veziak) [reply pkg/policy/policy.go]: fixed

File: pkg/github/github.go [hunk 2]
```diff
@@ -30,10 +30,29 @@ type GithubService struct {
 	Owner    string
 }
 
+func (svc *GithubService) GetUserTeams(organisation string, user string) ([]string, error) {
+	teamsResponse, _, _ := svc.Client.Teams.ListTeams(context.Background(), organisation, nil)
```
Comment (Spartakovic) [inline pkg/github/github.go]: should probably handle errors here

File: pkg/github/github.go [hunk 3]
```diff
@@ -30,10 +30,29 @@ type GithubService struct {
...
+		teamMembers, _, _ := svc.Client.Teams.ListTeamMembersBySlug(context.Background(), organisation, *team.Slug, nil)
+		for _, member := range teamMembers {
+			if *member.Login == user {
+				teams = append(teams, *team.Name)
+				break
+			}
+		}
+	}
+
+	for _, v := range teams {
```
Comment (Spartakovic) [inline pkg/github/github.go]: remove printing

File: pkg/github/github.go (hunk 2 above)
Comment (veziak) [reply pkg/github/github.go]: done

File: pkg/github/github.go (hunk 3 above)
Comment (veziak) [reply pkg/github/github.go]: done

File: pkg/digger/digger.go [hunk 4]
```diff
@@ -90,6 +104,7 @@ func RunCommandsPerProject(
 				if err != nil {
 					log.Printf("Error publishing comment: %v", err)
 				}
+				log.Println(msg)
```
Comment (Spartakovic) [inline pkg/digger/digger.go:107]: remove

Comment (veziak) [reply pkg/digger/digger.go:107]: that message is actually useful :) it duplicates what we post as comment, but it make clear why job failed in logs

Comment (Spartakovic) [reply pkg/digger/digger.go:107]: ah alright, we probably need to sort out usages of log, fmt, and println at some point

**Pull request merged**